### ISSUE TITLE
[Bug]: Overlay Trigger - cannot access overlay trigger content on iOS w/ VO

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3160]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

overlay-trigger

### Expected behavior

Should be able to access all of the content created with overlay triggers on all devices

### Actual behavior

Cannot access any of the slots from overlay triggers
"hover-content","long press-content" and "click-content" cannot be accessed when running VO

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. Go to https://studio.webcomponents.dev/preview/0QAwcRuCoUg7vd5qgdhr/src?branch=singleButton@7fBWg9Y3wmUwOKofZzrLDd2SRAr2&busid=2386c990-e54a-11ed-9e66-7bc22f9a2793
2. Use iPad
3. Turn on VO
4. Swipe until you get to the button
5. Note that it is not possible to get the "Search real hard..." text
6. Note that double tapping (to activate the click) seems to take me back to the "Sidebar" button in the browser chrome
7. Note that double tap and hold seems to have no way to activate the long press menu.

**Comment by najikahalsema** [comment]
Needs to be tested again since we have released Overlay v2.

**Comment by caseyisonit** [comment]
Hello @jnurthen in order to help us triage your issue, [can you please review our bug severity classification guide](https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md#issue-severity-classification) and provide us with the appropriate severity classification? Please update the label to the correct severity level and remove `SEV missing` label.

If we don't hear from you in 2 weeks, we will close this issue as not planned.

<img width="1024" height="1028" alt="Image" src="https://github.com/user-attachments/assets/d5fd78ee-6e8e-4967-97f3-4e4198b79f13" />

**Comment by castastrophe** [comment]
Thanks @jnurthen! We'll work on validating this against our latest updates and see if we can 🤞 close this out as fixed. 💃

**Comment by castastrophe** [comment]
I can't reproduce this with the new Overlay so I'm going to close this out but please do let us know if there are continued issues that you're seeing! Thank you!
